app/models/spree/product.rb

```diff
@@ -22,7 +22,12 @@ class Product < ApplicationRecord
     include LogDestroyPerformer
 
     self.belongs_to_required_by_default = false
-    self.ignored_columns += [:supplier_id, :variant_unit_scale, :variant_unit_name]
+    # These columns have been moved to variant. Currently this is only for documentation purposes,
+    # because they are declared as attr_accessor below, declaring them as ignored columns has no
+    # effect
```

Comment:
> declaring them as ignored columns has no effect

I think that the important role of `ignored_columns` comes in handy when the columns will be deleted. If you haven't ignored them with a previous deploy then the app raises errors after the data migration removing the columns and before the app has been reloaded. This allows for zero-downtime deployment.

Reply:
Thanks Maikel, we should probably update/remove the comment then to avoid future misunderstanding.

Reply:
Indeed, I just wanted to highlight that currently it has no effect. Do you think I should update the comment ?

Reply:
🤷 
Probably not worth the time thinking about it. Marking as resolved ;)